api/src/restful.js

```diff
@@ -15,9 +15,11 @@ export default function createRouter(db) {
   router.route('/reports').get(reports.get(reportsColl));
   router.route('/reports/current').get(reports.current(reportsColl));
   router.route('/reports').post(reports.create(reportsColl));
+  router.route('/reports/delete/:id').post(reports.remove(reportsColl));
 
   router.route('/updates').get(updates.get(updatesColl));
   router.route('/updates').post(updates.create(updatesColl));
+  router.route('/updates/delete/:id').post(updates.remove(updatesColl));
```

Comment:
Here you'll need to use the `delete` method:

```javascript
router.route('/updates/:id').delete(updates.remove(updatesColl));
```

And then probably write a new helper in `utils.js`:

```javascript
export function delete(url) {
  return fetch(url, {
    method: 'DELETE',
    // not sure if headers are required
  }).then(resp => resp.json());
}